File: tests/payloads/suite/python/kadane.py
import sys, random
n, seed = map(int, sys.stdin.readline().split())
random.seed(seed)
arr = [random.randint(-1000, 1000) for _ in range(n)]
cur = mx = arr[0]
for x in arr[1:]:
    cur = max(x, cur + x)
    mx = max(mx, cur)
print(mx)

File: tests/payloads/suite/java/kadane.java
import java.util.Scanner;
import java.util.Random;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        if (!sc.hasNextInt()) return;
        int n = sc.nextInt();
        int seed = sc.nextInt();
        
        Random random = new Random(seed);
        int[] arr = new int[n];
        for (int i = 0; i < n; i++) {
            arr[i] = random.nextInt(2001) - 1000;
        }
        
        long cur = arr[0];
        long mx = arr[0];
        
        for (int i = 1; i < n; i++) {
            int x = arr[i];
            cur = Math.max((long) x, cur + x);
            mx = Math.max(mx, cur);
        }
        
        System.out.println(mx);
    }
}
